src/Place/Canonical/DBALDuplicatePlaceRepository.php

```diff
@@ -78,4 +78,57 @@ public function getDuplicatesOfPlace(string $placeId): ?array
 
         return count($duplicates) > 0 ? $duplicates : null;
     }
+
+    /** @return ClusterRecord[] */
+    public function calculateNoLongerInCluster(): array
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb->select('dp.*')
+            ->from('duplicate_places', 'dp')
+            ->leftJoin('dp', 'duplicate_places_import', 'dpi', 'dpi.cluster_id = dp.cluster_id AND dpi.place_uuid = dp.place_uuid')
+            ->where('dpi.cluster_id IS NULL')
+            ->orderBy('dp.cluster_id', 'asc')
+            ->addOrderBy('dp.place_uuid', 'asc');
+
+        $statement = $qb->execute();
+
+        return $this->processRawToClusterRecord($statement->fetchAllAssociative());
+    }
+
+    public function countPlacesInDuplicatePlacesImport(string $placeId): int
```

Comment:
```suggestion
    public function countPlaceInDuplicatePlacesImport(string $placeId): int
```

Reply:
It counts how many placeS are in the import table so is the current name not correct?

Reply:
So that makes me wonder if `isPlaceInDuplicatePlacesImport(string $placeId): bool` is not the correct name?